Sources/AppcuesKit/Data/Models/Experience/ExperienceComponent.swift

```diff
@@ -280,6 +282,13 @@ extension ExperienceComponent {
         let width: Double
         let height: Double
     }
+
+    struct TextSpan: Decodable {
+        let text: String
+
+        /// Note: not all style properties can be applied to a `TextSpan`.
+        let style: Style
```

Comment:
`style` needs to be optional in light of the decision to allow inheritance of text styles from the root `style`.